src/SourceGenerators/Uno.UI.SourceGenerators/DependencyObject/DependencyPropertyGenerator.cs

```diff
@@ -175,8 +175,8 @@ private void GenerateAttachedProperty(IndentedStringBuilder builder, INamedTypeS
 			{
 				var propertyName = memberSymbol.Name.TrimEnd("Property", StringComparison.Ordinal);
 
-				var getMethodSymbol = ownerType.GetMethods().FirstOrDefault(m => m.Name == "Get" + propertyName);
-				var setMethodSymbol = ownerType.GetMethods().FirstOrDefault(m => m.Name == "Set" + propertyName);
+				var getMethodSymbol = ownerType.GetMethodsWithName("Get" + propertyName).FirstOrDefault();
```

Comment:
Not sure if that can be faster? (it avoids Linq completely)

```suggestion
                var getSymbols = ownerType.GetMembers("Get" + propertyName);
                IMethodSymbol getMethodSymbol = getSymbols.Length == 0 ? null : getSymbols[0] as IMethodSymbol;
```

Reply:
I'll probably make a `GetFirstMethodWithName`, since it's the usage in most locations

Reply:
Fixed in a separate PR